packages/protractor/src/protractor.conf.js

```diff
@@ -43,6 +44,49 @@ function setConf(conf, name, value, msg) {
   conf[name] = value;
 }
 
+/**
```

Comment:
how will this work in general, if we have special code to handle it for protractor and karma? does it make sense for cypress to be able to use rules_webtesting-provisioned browsers? maybe this code should be factored differently to serve all test runners, should we discuss (maybe with @DrMarcII)